menu_silver.rb

```diff
@@ -0,0 +1,21 @@
+adjectives = ['hot', 'cold', 'creamy', 'soft', 'hard', 'crunchy', 'warm', 'spicy', 'gelatinous',
```

Comment:
thanks for the note about this! We hadn't really learned that yet but I definitely see how it would be helpful!